src/libs/portal.ts

```diff
@@ -474,7 +454,6 @@ function parsePortalClasses(events: PortalCacheEvent[]) {
  * @returns A more-readable event title.
  */
 export function cleanUp(str: string) {
-	if (typeof str !== 'string') { return str; }
```

Comment:
Do we use the `cleanUp` function for anything that might possibly not be a string?

Reply:
From what I could tell, it was used on only calendar event summaries from our cache, which are all strings.